chat/avatar/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# @lg-chat/avatar
+
+## 2.0.6
+
+### Patch Changes
```

Comment:
Should we make a note in the CHANGELOGs that previous to version X.Y.Z the package was private?

Reply:
They say "First public release" :)